remotespark/livyclientlib/livysession.py

```diff
@@ -12,7 +13,8 @@
 from .livyreliablehttpclient import LivyReliableHttpClient
 
 
-class LivySession(object):
+
+class LivySession(GuidMixin, object):
```

Comment:
From what I read, you DO need to call it, no?
http://stackoverflow.com/a/1385836